pkgs/tools/security/ghidra/extensions/ghidraninja-ghidra-scripts/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchFromGitHub
+, buildGhidraScripts
+, binwalk
+, swift
+, yara
+}:
+
+buildGhidraScripts {
+  pname = "ghidraninja-ghidra-scripts";
+  version = "unstable-2020-10-07";
+
+  src = fetchFromGitHub {
+    owner = "ghidraninja";
+    repo = "ghidra_scripts";
+    rev = "99f2a8644a29479618f51e2d4e28f10ba5e9ac48";
+    sha256 = "aElx0mp66/OHQRfXwTkqdLL0gT2T/yL00bOobYleME8=";
+  };
+
+  postPatch = ''
+    # Replace subprocesses with store versions
+    substituteInPlace binwalk.py --replace-fail 'subprocess.call(["binwalk"' 'subprocess.call(["${binwalk}/bin/binwalk"'
+    substituteInPlace swift_demangler.py --replace-fail '"swift"' '"${swift}/bin/swift"'
+    substituteInPlace yara.py --replace-fail 'subprocess.check_output(["yara"' 'subprocess.check_output(["${yara}/bin/yara"'
+    substituteInPlace YaraSearch.py --replace-fail '"yara "' '"${yara}/bin/yara "'
+  '';
+
+  meta = with lib; {
```

Comment:
Should those have a maintainer added? I'm personally not very interested in maintaining all the plugins - I don't follow their development nor use them.

Reply:
From what I understand, [this line](https://github.com/NixOS/nixpkgs/pull/310343/files#diff-27c8b6de51fbcd9db56d2fd155d2f57b4220b45de42917ccf8d06906474ca926R12) would make us co-maintainers. Since you don't want that, I'll make myself the only maintainer. However, I might remove all plugins not shipped by Ghidra itself, since I don't use those either.

Reply:
Nvm, this line makes me + whoever registers in an individual plugin a maintainer.